sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamSqlUnparseContext.java

```diff
@@ -110,6 +111,26 @@ public SqlNode toSql(RexProgram program, RexNode rex) {
     public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
       writer.literal("DATETIME '" + timestampString.toString() + "'");
     }
+
+    @Override
```

Comment:
@ZijieSong946 This class is a good candidate for AutoValue to eliminate this boilerplate.
(This got missed in #12348 because it was merged without a Jenkins run. cc: @robinyqiu )

Reply:
Ah, my bad. I only run spotless and checkStyle locally but not spotBugs.

Reply:
Got it. Thanks for fixing that.